web/templates/symptom_tracker/new.html.eex

```diff
@@ -3,7 +3,7 @@
 
   <%= form_for @changeset, symptom_tracker_path(@conn, :create), fn f -> %>
     <div class="">
-      <%= label f, "Rate how much this problem or difficulty affected you today:", class: "f5 b db mb2" %>
+      <%= label f, "Rate how much this problem or difficulty effected you today:", class: "f5 b db mb2" %>
```

Comment:
_fairly_ certain the context of this sentence suggests that `affected` was _correct_ ...
http://www.diffen.com/difference/Affected_vs_Effected
_clarification_ sought:
https://github.com/healthlocker/healthlocker/issues/560#issuecomment-303691708